sphire/protocols/protocol_cryolo_picking.py

```diff
@@ -137,6 +137,13 @@ def _defineParams(self, form):
                             " crYOLO can use multiple GPUs - in that case"
                             " set to i.e. *0 1 2*.")
 
+        form.addParam('boxSizeFactor', params.FloatParam,
```

Comment:
I don't really like to add unnecessary parameters, specially if not in the original program.

Reply:
Additionally, it is changing the current default behaviour. I think this is not necessary and the box size can be set in the extraction. And if the idea is to do it automatically for extraction, maybe 1.5 is still too low, but this logic should be implemented better somewhere else (i.e automator script or something like that)

Reply:
Cryolo boxsize estimation is very tight. If we register that box size we will need to increase it later on at extraction. For automated workflows, this change will help to avoid guessing the boxsize in advance. 

I actually think that adding parameters to protocols that are not in the original program are an added value if they do so, of course.

Reply:
this plugin does not produce boxsize in streaming mode

Reply:
@pconesa I agree if the added value is clear...it is not to me in this case...if you are able to guess a factor (for me 1.5 seems still low, but I don't think there is a general one for all cases), then you are able to guess the boxsize and you can already set that. I don't really see the need of this "addition"

Reply:
![image](https://user-images.githubusercontent.com/785633/141980491-a9542df0-5a68-413a-9029-98c5078feb8f.png)
This is default boxsize estimated by cryolo. I do not think this boxsize is suitable for extraction. The idea of this parameter is to have a way to increase it by factor. I can agree that 1.5 might not be the best default value. I could even agree to have 1 by default. But I think it is useful to have it there.

Reply:
@azazellochg , you mean the output boxSize? the Integer one?

Reply:
As you wish @pconesa , please keep at least 1 as default and avoid float multiplication by 1.0

Reply:
I'll keep the default value at 1 and test if factor is 1.0. Do you think is there a much difference between the performance of a float multiplication and an if test (if value != 1)?

Reply:
@pconesa yes. https://github.com/scipion-em/scipion-em-sphire/issues/45

Reply:
Should do it now, thanks @azazellochg